src/routes/patches/PatchItem.svelte

```diff
@@ -79,6 +83,7 @@
 						alt="dropdown"
 					/>
 				</Button>
+				</li>
```

Comment:
Why is this a list item?

Reply:
Because it's a direct child of the list

Reply:
I don't understand why that means it should be wrapped in li, but even then please fix the formatting if this change is necessary

Reply:
> I don't understand why that means it should be wrapped in li

Why:

> Screen readers have a specific way of announcing lists. This feature makes lists clearer to understand, but will only work if lists are properly structured.
> 
> When content elements other than list items are contained within a set of list elements, screen readers cannot inform the listener that they are listening to items within the list.
> 
> For a list to be valid, it must have both parent elements (a set of `ul` elements or a set of `ol` elements) and child elements (**declared inside of these tags using the `li` element**), and any other content elements are invalid.
> 
> Although some non-content elements such as script, template, style, meta, link, map, area, and datalist are permitted within lists, **content elements other than `li` are not permitted.**

Reply:
I fixed the formatting,  my bad.
@KTibow answer explains why I've made that change.